terraform/schema.yaml

```diff
@@ -458,6 +459,19 @@ variables:
                 - ${subnet_span}
                 - "Regional Subnet"
 
+  place_all_compute_in_same_ad:
+    visible:
+      and:
+        - ${orm_create_mode}
+        - eq:
+            - ${subnet_span}
+            - "Regional Subnet"
+    type: boolean
+    required: true
+    default: false
+    title: "Use Same Availability Domain for all Compute Instances"
+    description: "Use the same Availability Domain for all Compute instances as that of the WebLogic Administration Server's Compute instance."
```

Comment:
For clarity, add to the description - what happens if this option is not selected: Each compute instance will be created in a separate AD (AD rotation). 

Same comment for updating the description field of the .tf files

Reply:
Updated the files